src/main/java/hudson/plugins/performance/reports/UriReport.java

```diff
@@ -190,8 +194,8 @@ public long getAverage() {
     }
 
     private long getDurationAt(double percentage) {
-        if (percentage < 0 || percentage > 1) {
-            throw new IllegalArgumentException("Argument 'percentage' must be a value between 0 and 1 (inclusive)");
+        if (percentage < ZERO_PERCENT || percentage > ONE_HUNDRED_PERCENT) {
```

Comment:
Original constant are floats up to 1, new constants are up to 100. Isnt there error?
BTW I'd suggest to keep float approach, since users might want float percentiles in the future, like 99.5 or 99.9